test/e2e/validation_test.go

```diff
@@ -464,6 +465,64 @@ func testExpectedServicesRunning(t *testing.T) {
 	}
 }
 
+// testServicesConfigMap tests 2 aspects of expected functionality for the services ConfigMap
+// 1. It exists on operator startup and 2. It is recreated when deleted
+func testServicesConfigMap(t *testing.T) {
+	tc, err := NewTestContext()
+	require.NoError(t, err)
+
+	operatorVersion, err := getWMCOVersion()
+	require.NoError(t, err)
+	// There is no easy way to populate ldflags when running e2e tests without having to maintain the WMCO version in
+	// another location, so that version.Get() will return something other than "". Because of this,
+	// we cannot use servicescm.Name and must build to the services CM name by retrieving the version in another manner.
+	servicesConfigMapName := servicescm.NamePrefix + operatorVersion
+
+	err = tc.validateWindowsServicesConfigMap(servicesConfigMapName)
+	require.NoErrorf(t, err, "error validating ConfigMap: %s", servicesConfigMapName)
+
+	err = tc.deleteAndWaitForServicesConfigMap(servicesConfigMapName)
+	require.NoErrorf(t, err, "error ensuring ConfigMap %s is re-created when deleted", servicesConfigMapName)
+}
+
+// validateWindowsServicesConfigMap ensures the windows-services ConfigMap exists in the cluster and has valid contents
+func (tc *testContext) validateWindowsServicesConfigMap(cmName string) error {
+	windowsServices, err := tc.client.K8s.CoreV1().ConfigMaps(tc.namespace).Get(context.TODO(),
+		cmName, meta.GetOptions{})
+	if err != nil {
+		return errors.Wrapf(err, "error retrieving ConfigMap: %s", cmName)
+	}
+
+	if _, err := servicescm.Parse(windowsServices.Data); err != nil {
+		return errors.Wrapf(err, "invalid %s ConfigMap data", cmName)
+	}
```

Comment:
I again see no point  calling this function here. You are using the same code to test if the ConfigMap is created correctly. There is no way this can be incorrect. I think checking if the ConfigMap is present is sufficient. 

The more interesting test would be to stop the operator, create an invalid ConfigMap and ensure it is recreated correctly.